Examples/CodePushDemoApp/android/app/react.gradle

```diff
@@ -0,0 +1,75 @@
+def config = project.hasProperty("react") ? project.react : [];
+
+def bundleAssetName = config.bundleAssetName ?: "index.android.bundle"
+def entryFile = config.entryFile ?: "index.android.js"
+
+// because elvis operator
```

Comment:
Remove references to Elvis operator.

You're using the ternary operator below (line 8).

http://groovy-lang.org/operators.html#_elvis_operator